WorkflowSwiftUIExperimental.podspec

```diff
@@ -18,8 +18,8 @@ Pod::Spec.new do |s|
 
     s.source_files = 'WorkflowSwiftUIExperimental/Sources/*.swift'
 
-    s.dependency 'Workflow', WORKFLOW_VERSION
-    s.dependency 'WorkflowUI', WORKFLOW_VERSION
+    s.dependency 'Workflow', "~> #{WORKFLOW_MAJOR_VERSION}.0"
+    s.dependency 'WorkflowUI', "~> #{WORKFLOW_MAJOR_VERSION}.0"
```

Comment:
Say a new major version of Workflow comes out at 4.0.0, seems like that's only when you'd want this dependency to change? So you'd wanna ignore things like 3.6.1, 3.7.0 as well as 4.1.0, 4.1.1, etc? Just wanna make sure I understand the intention here.

Reply:
yeah, this change means that version 0.2 of this pod requires version 3.x.y of its two dependencies, but any one will do.